README.markdown

```diff
@@ -205,7 +205,7 @@ open-vm-tools is available with these operating systems:
 
 ## Development
 
-Please see [CONTRIBUTING.md](CONTRIBUTING.md) for information on how to contribute.
+Please see [Vox Populi Documentation](https://voxpupuli.org/docs/) for information on how to contribute.
```

Comment:
Hi @pillarsdotnet the link should be https://github.com/voxpupuli/puppet-openvmtools/blob/master/.github/CONTRIBUTING.md

Reply:
Thanks; fixed.